plan.go

```diff
@@ -134,3 +138,27 @@ func (s *plans) Logs(ctx context.Context, planID string) (io.Reader, error) {
 		logURL: u,
 	}, nil
 }
+
+
+// Provide the JSON version of the plan from the TFE API.
+func (s *plans) JsonOutput(ctx context.Context, planID string) ([]byte, error) {
+	if !validStringID(&planID) {
+		return nil, errors.New("invalid value for plan ID")
+	}
+
+	u := fmt.Sprintf("plans/%s/json-output", url.QueryEscape(planID))
+	// TODO: FInd a way to not have to use our own code to do this - maybe extend the API?
```

Comment:
I'm not quite sure I understand what this TODO comment is suggesting - could you clarify?

Reply:
Sorry, it was left over from before, when I was doing it a different way.  I've removed it now.